src/module-vsbridge-indexer-catalog/Model/Indexer/DataProvider/Product/ParentData.php

```diff
@@ -0,0 +1,93 @@
+<?php
+/**
+ * @package  Divante\VsbridgeIndexerCatalog
+ * @author Marcin Dykas <mdykas@divante.pl>
+ * @copyright 2019 Divante Sp. z o.o.
+ * @license See LICENSE_DIVANTE.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Divante\VsbridgeIndexerCatalog\Model\Indexer\DataProvider\Product;
+
+use Divante\VsbridgeIndexerCore\Api\DataProviderInterface;
+use Magento\Catalog\Api\ProductRepositoryInterface;
+use Magento\GroupedProduct\Model\Product\Type\Grouped as GroupedType;
+use Magento\ConfigurableProduct\Model\Product\Type\Configurable as ConfigurableType;
+use Magento\Bundle\Model\Product\Type as BundleType;
+use Divante\VsbridgeIndexerCatalog\Model\Product\ParentResolver;
+
+/**
+ * Class ParentData
+ */
+class ParentData implements DataProviderInterface
+{
+    const FIELD_NAME = 'parent_sku';
+
+    /**
+     * @var GroupedType
+     */
+    private $groupedType;
+
+    /**
+     * @var ConfigurableType
+     */
+    private $configurableType;
+
+    /**
+     * @var BundleType
+     */
+    private $bundleType;
+
+    /**
+     * @var ParentResolver
+     */
+    private $parentResolver;
+
+    /**
+     * ParentData constructor.
+     *
+     * @param GroupedType $groupedType
+     * @param ConfigurableType $configurableType
+     * @param BundleType $bundleType
+     * @param ParentResolver $parentResolver
+     */
+    public function __construct(
+        GroupedType $groupedType,
+        ConfigurableType $configurableType,
+        BundleType $bundleType,
+        ParentResolver $parentResolver
+    ) {
+        $this->groupedType = $groupedType;
+        $this->configurableType = $configurableType;
+        $this->bundleType = $bundleType;
+        $this->parentResolver = $parentResolver;
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function addData(array $indexData, $storeId)
+    {
+        $parentIds = [];
+
+        foreach ($indexData as $productId => $productData) {
```

Comment:
As it's not use by default VSF yet, so exporting parent sku - should be optional.
Please add an option to configuration - so user can choose if we should export ParentSku or not - by default set it to NO.

Reply:
Fixed